### ISSUE TITLE
great stuff - please teach us to adopt it to experimental fluid dynamics

### ISSUE THREAD

**Issue by alexlib** [acrlakshman/twitter-bot-computational-fluids#1]
Please, some instructions. Do we need to run AWS or GCP to have it alive? how do we modify the keywords?

**Comment by acrlakshman** [comment]
Hi @alexlib, for this application, we could use either some service providers or simply run it on our machines. The way I did is, I have server space on digitalocean and I have docker installed there. All I did is, cloned this repo onto my server and deployed it to docker swarm using exactly the instructions I provided in README file. But before this, we need to have developer account approved on twitter to get API keys. Though the background details are available on any blog posts, for the sake of completeness, I will write detailed instructions on how to get things running and link it to this repo. I will update you soon.
